patches/llvm-all/linux-next/arm64/silence-dwarf2-warnings.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Makefile b/Makefile
+index 9f9015664..ec3be353c 100644
+--- a/Makefile
++++ b/Makefile
+@@ -810,7 +810,6 @@ DEBUG_CFLAGS	+= -gsplit-dwarf
+ else
+ DEBUG_CFLAGS	+= -g
+ endif
```

Comment:
this should be an `ifndef` with `LLVM_IAS` (probably pretty close to the other reference to `LLVM_IAS` in this file) rather than deleting it.